base/shell/explorer/traywnd.cpp

```diff
@@ -660,8 +660,18 @@ class CTrayWindow :
         return S_OK;
     }
 
+    void SaveState()
+    {
+        if (SHRestricted(REST_NOSAVESET))
+            return ;
```

Comment:
666 - the number of the beast:
```suggestion
            return;
```